mkdocs.yaml

```diff
@@ -0,0 +1,117 @@
+site_name: Deeporigin CLI and Python Client
+site_author: Deep Origin
+repo_url: https://github.com/formiclabs/deeporigin
+repo_name: formiclabs/deeporigin
```

Comment:
The repo name is `formiclabs/deeporigin-client` (`-client`) is missing.

Reply:
fixed